packages/arb-validator-core/utils/rpc.go

```diff
@@ -0,0 +1,19 @@
+package utils
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [the file doesn't have an appropriate header](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)